chalice/app.py

```diff
@@ -299,7 +309,7 @@ def __init__(self, body, headers=None, status_code=200):
 
     def to_dict(self, binary_types=None):
         body = self.body
-        if not isinstance(body, (str, bytes)):
+        if not isinstance(body, _STRING_TYPES):
```

Comment:
Can you rename this constant?  It took me a while to figure this out because I was assuming this mapped to `six.string_types` which doesn't include the bytes type in either py2 or py3.  mypy using `AnyStr` for this type so maybe that would work?